src/en/dynasty/src/eu/kanade/tachiyomi/extension/en/dynasty/Dynasty.kt

```diff
@@ -0,0 +1,698 @@
+        if (response.request.url.pathSegments[0] == "chapters") {
+            return chapterDetailsParse(response)
+        }
+
+        val data = response.parseAs<MangaResponse>()
+
+        val authors = LinkedHashSet<String>()
+        val tags = LinkedHashSet<String>()
+        val others = LinkedHashSet<Pair<String, String>>()
+        val publishingStatus = LinkedHashSet<String>()
+
+        data.tags.forEach { tag ->
+            when (tag.type) {
+                "Author" -> authors.add(tag.name)
+                "General" -> tags.add(tag.name)
+                "Status" -> {
+                    publishingStatus.add(tag.name)
+                    others.add(tag.type to tag.name)
+                }
+                else -> others.add(tag.type to tag.name)
+            }
+        }
+
+        data.taggings.filterIsInstance<MangaChapter>().forEach { tagging ->
+            tagging.tags.forEach { tag ->
+                when (tag.type) {
+                    "Author" -> authors.add(tag.name)
+                    "General" -> tags.add(tag.name)
+                    "Series", "Scanlator", "Doujin", "Anthology" -> {}
+                    else -> others.add(tag.type to tag.name)
+                }
+            }
+        }
+
+        return SManga.create().apply {
+            title = data.name
+            author = authors.joinToString()
+            artist = author
+            description = buildString {
+                val prefChapterFetchLimit = preferences.chapterFetchLimit
+                if (prefChapterFetchLimit < data.totalPages) {
+                    append("IMPORTANT: Only first $prefChapterFetchLimit pages of chapter list will be fetched. You can change this in extension settings.\n\n")
+                }
+
+                data.description?.let {
+                    val desc = Jsoup.parseBodyFragment(
+                        decodeUnicode(it),
+                    )
+                    desc.select("a").remove()
+
+                    append(desc.wholeText().trim())
+                    append("\n\n")
+                }
+
+                append("Type: ", data.type, "\n\n")
+
+                for ((type, values) in others.groupBy { it.first }) {
+                    append(type, ":\n")
+                    values.forEach { append("• ", it.second, "\n") }
+                    append("\n")
+                }
+                if (data.aliases.isNotEmpty()) {
+                    append("Aliases:\n")
+                    data.aliases.forEach { append("• ", it, "\n") }
+                    append("\n")
+                }
+            }.trim()
+            genre = tags.joinToString()
+            status = when {
+                publishingStatus.contains("Ongoing") -> SManga.ONGOING
+                publishingStatus.contains("Completed") -> SManga.COMPLETED
+                publishingStatus.contains("On Hiatus") -> SManga.ON_HIATUS
+                // publishingStatus.contains("Licensed") -> SManga.LICENSED
```

Comment:
Why? AFAIK the previous guard blocking updates for licensed entries was removed ages ago.